abnormal-security/info.json

```diff
@@ -0,0 +1,1259 @@
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Threat ID",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "threatId",
+          "tooltip": "Specify the UUID of the threat.",
+          "description": "Specify the UUID of the threat."
+        },
+        {
+          "title": "Action ID",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "actionId",
+          "tooltip": "Specify the UUID representing the action ID for a threat.",
+          "description": "Specify the UUID representing the action ID for a threat."
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {
+        "status": "",
+        "description": ""
+      }
+    },
+    {
+      "operation": "download_data_from_threat_log",
+      "annotation": "download_data_from_threat_log",
+      "category": "investigation",
+      "title": "Download Data from Threat Log",
+      "description": "Download data from Threat Log in .csv format.",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Filter",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "filter",
+          "tooltip": "Specify the filter, currently the only key that is supported is receivedTime. Datetime string must follow this format YYYY-MM-DDTHH:MM:SSZ. Range is limited to 14 days.",
+          "description": "Specify the filter, currently the only key that is supported is receivedTime. Datetime string must follow this format YYYY-MM-DDTHH:MM:SSZ. Range is limited to 14 days."
+        },
+        {
+          "title": "Source",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "select",
+          "name": "source",
+          "tooltip": "Filters threats based on the source of detection. Default value is all.",
+          "description": "Filters threats based on the source of detection. Default value is all.",
+          "options": [
+            "all",
+            "advanced"
+          ]
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {}
```

Comment:
If the output of action is in csv format, Shouldn't we upload it to FSR Attachment module and output it's IRI info?

Reply:
done